Makefile

```diff
@@ -74,7 +74,7 @@ UNAME_M := $(shell uname -m)
 
 BUILD_IMAGE := quay.io/stackrox-io/apollo-ci:$(shell sed 's/\s*\#.*//' BUILD_IMAGE_VERSION)
 ifneq ($(UNAME_M),x86_64)
-	BUILD_IMAGE = docker.io/library/golang:$(shell cat EXPECTED_GO_VERSION | cut -c 3-)
+	BUILD_IMAGE = quay.io/projectquay/golang:1.21
```

Comment:
1. Yes, https://quay.io/repository/projectquay/golang?tab=tags&tag=latest says 1.21 is published for linux on arm64 but I'm afraid you don't have a way to check its functioning since you don't have ARM-based Mac. Shall we keep `docker.io/library/golang:1.21` instead given it's a dev-time dependency?
2. Are there instructions how to update Go version for ACS? Bumping a tag on this line should become a step in these instructions.

Reply:
> I'm afraid you don't have a way to check its functioning since you don't have ARM-based Mac.
I don't have a way to check if docker is functioning too :cry: 
 
 1. Changed to docker
 2. Added info https://docs.engineering.redhat.com/display/StackRox/How+to+Bump+Golang+Version

Reply:
I found https://docs.engineering.redhat.com/display/StackRox/How+to+Bump+Golang+Version thin on content and in some points misleading. Would be nice if you improve it given the exercise you do now.

Reply:
Me too, on one hand I can update it but on the other since we have Go 1.21 with toolchain support maybe we can remove everything from it as go upgrade will be just the matter of upgrading toolchain

Reply:
I wish we could erase downstream from it, but we can't. Also, updating rox-ci-image for OSCI was far from trivial IIRC and having at least pointers is very helpful. I vote to keep and improve rather than deleting.